detection-rules/link_fake_security_login_alert_generic.yml

```diff
@@ -0,0 +1,101 @@
+name: "Credential Phishing: Fake Security Login Alert (Generic)"
+description: "Detects phishing attempts that claim suspicious login activity or unusual sign-in attempts regardless of brand"
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  
+  // Subject contains security alert language
+  and regex.icontains(subject.subject,
+                     "unusual login|unusual sign|login attempt|sign.?in attempt|suspicious sign|unusual activity|security alert")
+  
+  // Body contains security alert language
+  and regex.icontains(body.current_thread.text,
+                     "detect.*login|unusual.*login|suspicious.*login|login attempt|unusual sign|IP Address|Location|don't recognize|security|verify.*account|suspicious activity")
+  
+  // Link with suspicious display text
+  and any(body.links,
+         regex.icontains(.display_text, 
+                        "sign in|log in|login|verify|continue|account|confirm")
+  )
+  
+  // Link domains don't match sender
+  and any(body.links,
+         .href_url.domain.root_domain != sender.email.domain.root_domain
+         and .href_url.domain.root_domain not in $org_domains
+  )
+  
+  // Check for NLU credential theft intent
+  and any(ml.nlu_classifier(body.current_thread.text).intents,
+          .name == "cred_theft" and .confidence in ("medium", "high")
+  )
+  
+  // Explicit negations for known legitimate security alert senders
+  and not (
+    regex.icontains(sender.email.domain.domain, 
+                   "lastpass.com|teachable.com|accountprotection.microsoft.com|notificationservices.org")
```

Comment:
This is probably not how we want to list domains (usually `not in` reads better)